docs/src/pages/docs/installation/configuring.mdx

```diff
@@ -113,6 +113,8 @@ RequestHeader set X-Forwarded-Proto "https"
 Beyond FAB supported providers (Github, Twitter, LinkedIn, Google, Azure, etc), its easy to connect
 Superset with other OAuth2 Authorization Server implementations that support “code” authorization.
 
+Make sure the pip package `Authlib` is installed on the webserver.
```

Comment:
Can we link to the pip package?

Reply:
addressed in https://github.com/apache/superset/pull/15010/commits/bb2253e5a541af60b8777519f401837e27310a22